test/test_operations.py

```diff
@@ -1298,6 +1298,303 @@ def test_git_revisons(self):
     self.assertTrue(revs['xla'])
     self.assertTrue('torch' in revs)
 
+class TestTypePromotion(TestCase):
```

Comment:
This is 4 spaces indented ... I am afraid passing it with `google-pyformat` might make a mess.
Can you try to bring it to 2 spaces manually?